Mathlib/NumberTheory/Padics/PadicNorm.lean

```diff
@@ -305,6 +305,17 @@ theorem nat_lt_one_iff (m : ℕ) : padicNorm p m < 1 ↔ p ∣ m := by
   rw [← Int.coe_nat_dvd, ← int_lt_one_iff, Int.cast_ofNat]
 #align padic_norm.nat_lt_one_iff padicNorm.nat_lt_one_iff
 
+lemma _root_.Rat.isInt.eq_num {q : ℚ} (h : q.isInt) : q = q.num := by
+  rw [Rat.isInt, Nat.beq_eq_true_eq] at h
+  exact (Rat.coe_int_num_of_den_eq_one h).symm
```

Comment:
This lemma should be moved earlier.

Reply:
Can you suggest a place this could go? The most natural place I can think of is `Std/Data/Rat`, but that seems like its a different package(?)

Reply:
Perhaps here? But this seems barren so I don't know: https://github.com/leanprover-community/mathlib4/blob/master/Mathlib/Init/Data/Rat/Basic.lean

Reply:
I would suggest Data.Rat.Defs, since that is where `Rat.coe_int_num_of_den_eq_one` is located

Reply:
OK done.